src/panels/lovelace/common/compute-notifications.ts

```diff
@@ -1,8 +1,8 @@
-import { HassEntity } from "home-assistant-js-websocket";
+import { HassEntities } from "home-assistant-js-websocket";
 
 import computeDomain from "../../../common/entity/compute_domain";
 
-export const computeNotifications = (states: HassEntity[]): string[] => {
+export const computeNotifications = (states: HassEntities[]): string[] => {
```

Comment:
`HassEntities`, not `HassEntities[]`. The `[]` indicate that it's a list of that type. HassEntities is an object with as keys entity_id and as value the state (it's `hass.states`)